service/controller/aws/v15/resource/certconfigcleansing/resource.go

```diff
@@ -0,0 +1,107 @@
+package certconfigcleansing
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	Name = "certconfigcleansingv1"
+)
+
+// Config represents the configuration used to create a new certconfigcleansing resource.
+type Config struct {
+	G8sClient versioned.Interface
+	Logger    micrologger.Logger
+}
+
+// Resource implements the certconfigcleansing resource.
+type Resource struct {
+	g8sClient versioned.Interface
+	logger    micrologger.Logger
+}
+
+func (r Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
```

Comment:
I was thinking about splitting this function to different file but after all, I will delete this resource.

Reply:
I think we should try to stick to our high standards regardless. As the saying goes, how you do anything is how you do everything.